syntax = "proto2";

package crypto;
import "protos/utils.proto";

message SetMsgRequest {
    required bytes login = 1;
    required bytes msg = 2;
}

message SetMsgResponse {
    required bytes msg = 1;
}

extend Option {
    optional SetMsgRequest SET_MSG_REQUEST = 11;
    optional SetMsgResponse SET_MSG_RESPONSE = 12;
}